src/main/java/longah/node/Member.java

```diff
@@ -82,4 +82,13 @@ public String toString() {
     public String getName() {
         return this.name;
     }
+
+    /**
+     * Used to check whether the input String matches the name of a member.
+     * @param memberName String representation of a member name
+     * @return A boolean value checking whether the input matches with name.
+     */
+    public boolean isEqual(String memberName) {
```

Comment:
Could improve method naming to something more meaningful like `isName` instead.

Reply:
Done!